src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -40,20 +44,19 @@ enum Spells
     SPELL_FEAR                  = 68950,
     SPELL_SOULSTORM             = 68872,
     SPELL_SOULSTORM_CHANNEL     = 69008,    // pre-fight
-    SPELL_SOULSTORM_VISUAL      = 68870,    // pre-cast soulstorm
+    SPELL_SOULSTORM_VISUAL      = 68870,    // pre-cast Soulstorm
     SPELL_PURPLE_BANISH_VISUAL  = 68862     // Used by Soul Fragment (Aura)
 };
 
-enum Events
+enum Text
 {
-    EVENT_MAGIC_BANE    = 1,
-    EVENT_SHADOW_BOLT   = 2,
-    EVENT_CORRUPT_SOUL  = 3,
-    EVENT_SOULSTORM     = 4,
-    EVENT_FEAR          = 5,
+    SAY_AGGRO           = 0,
+    SAY_KILL            = 1,
+    SAY_SOUL_STORM      = 2,
+    SAY_CORRUPT_SOUL    = 3
```

Comment:
Did you check it in the database to see if these values match the data expected to be used? In the last author's PR it was wrong